packages/create-rspack/utils/index.js

```diff
@@ -0,0 +1,28 @@
+const fs = require("fs");
+const path = require("path");
+
+/**
+ * format the projectName
+ * @param {*} targetDir
+ * @returns
+ */
+exports.formatTargetDir = function (targetDir) {
+	return targetDir.trim().replace(/\/+$/g, "");
+};
+
+/**
+ * Empty the folder (for the case of duplicate folder name)
+ * @param dir
+ * @returns
+ */
+exports.emptyDir = function (dir) {
+	if (!fs.existsSync(dir)) {
+		return;
+	}
+	for (const file of fs.readdirSync(dir)) {
+		if (file === ".git") {
+			continue;
+		}
+		fs.rmSync(path.resolve(dir, file), { recursive: true, force: true });
+	}
```

Comment:
```suggestion
	for (const file of fs.readdirSync(dir)) {
		fs.rmSync(path.resolve(dir, file), { recursive: true, force: true });
	}
```

To determine if the `.git` folder is unnecessary, you should check if this folder is empty (i.e., only containing .git), and use the `break` keyword instead of `continue` to exit the loop. Alternatively, you can write an `isEmpty()` function.

```js
function isEmpty(path) {
  const files = fs.readdirSync(path)
  return (
    files.length === 0 ||
    (files.length <= 2 &&
      files.every((f) => f === '.git' || f.toUpperCase() === 'LICENSE'))
  )
}
```